src/store/cart.js

```diff
@@ -0,0 +1,21 @@
+let initialState = {
+    cart: [],
+};
+
+const cartReducer = (state = initialState, action) => {
+    switch (action.type) {
+        case 'ADD_TO_CART':
+            return {
+                cart: [...state.cart, action.payload],
+            }
+            case 'REMOVE_FROM_CART':
+                return {
+                    ...state,
+                    cart: state.cart.filter(item => item.name !== action.payload.name),
+                }
```

Comment:
If we have more than one of the same item, trying to remove one removes all of them. We want to remove only the one we clicked on.